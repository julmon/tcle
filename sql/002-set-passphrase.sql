SET search_path TO public, s1;
SELECT tcle_set_passphrase('my secret passphrase');
