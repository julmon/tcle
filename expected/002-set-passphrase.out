SET search_path TO public, s1;
SELECT tcle_set_passphrase('my secret passphrase');
 tcle_set_passphrase 
---------------------
 t
(1 row)

